assets/src/scripts/components/hydrograph/audible.js

```diff
@@ -46,28 +46,30 @@ export const createSound = memoize(/* eslint-disable no-unused-vars */ tsKey =>
 });
 
 export const updateSound = function ({enabled, points}) {
-    const audioCtx = getAudioContext();
-    for (const tsKey of Object.keys(points)) {
-        const point = points[tsKey];
-        const {compressor, oscillator, gainNode} = createSound(tsKey);
-
-        compressor.threshold.setValueAtTime(-50, audioCtx.currentTime);
-        compressor.knee.setValueAtTime(40, audioCtx.currentTime);
-        compressor.ratio.setValueAtTime(12, audioCtx.currentTime);
-        compressor.attack.setValueAtTime(0, audioCtx.currentTime);
-        compressor.release.setValueAtTime(0.25, audioCtx.currentTime);
-
-        oscillator.frequency.setTargetAtTime(
-            enabled && point ? point : null,
-            audioCtx.currentTime,
-            .2
-        );
-
-        gainNode.gain.setTargetAtTime(
-            enabled && point ? volumeScale(point) : null,
-            audioCtx.currentTime,
-            .2
-        );
+    if (enabled) {
```

Comment:
I think this might prevent the sound from being turned off if the audible checkbox is clicked while the sound is playing. Could you check if AudioContext == null instead?

Reply:
I put this in to prevent the sound from playing on Firefox and Safari when the page was loaded. When clicking the toggle, the sound does stop.

Reply:
Huh - I'm surprised, but sounds good.